hazelcast-hibernate5/src/main/java/com/hazelcast/hibernate/local/CleanupService.java

```diff
@@ -29,13 +29,22 @@
  */
 public final class CleanupService {
 
-    private static final long FIXED_DELAY = 60;
-    private static final long FIXED_DELAY1 = 60;
+    /**
+     * Default fixed delay in seconds for scheduled job.
+     * Visible for testing.
```

Comment:
Where do you use it in tests? Could it be `private`?

Reply:
You're right, it's not used now. Updated to private.